examples/guides/testing/running-all-the-test-modules/README.md

```diff
@@ -0,0 +1,121 @@
+# How to run all the test modules?
```

Comment:
Shall we move this md file into the docs directory, since these are instructions to the developers using this repository?

Reply:
Done.